lambda/src/main/scala/pricemigrationengine/handlers/NotificationEmailHandler.scala

```diff
@@ -58,9 +60,9 @@ object NotificationEmailHandler {
                 billing_postal_code = postalCode,
                 billing_state = contact.MailingAddress.state,
                 billing_country = country,
-                payment_amount = newPrice,
+                payment_amount = estimatedNewPrice,
                 next_payment_date = startDate,
-                payment_frequency = billingPeriod,
+                payment_frequency = s"${billingPeriod}ly",
```

Comment:
I think there's a `Semi_Annual` billing period, but will need to check that.  If so, might be better formatted as `semiannually`.

Reply:
Ill put in a mapping and fail if we see any values not in the mapping table.

Reply:
👍